static/app/components/onboardingWizard/taskConfig.tsx

```diff
@@ -428,6 +433,14 @@ export function getOnboardingTasks({
   ];
 }
 
+const replayRelatedTasks = [OnboardingTaskKey.SESSION_REPLAY];
+const performanceRelatedTasks = [
+  OnboardingTaskKey.FIRST_TRANSACTION,
+  OnboardingTaskKey.PERFORMANCE_GUIDE,
+  OnboardingTaskKey.METRIC_ALERT,
+];
+const feedbackRelatedTasks = [OnboardingTaskKey.USER_REPORTS];
+
 export function getMergedTasks({organization, projects, onboardingContext}: Options) {
   const taskDescriptors = getOnboardingTasks({organization, projects, onboardingContext});
```

Comment:
I'm trying to mock this to only return a constant set that I'm giving to it so I can test better the filtering functionality that I'm adding below.

Reply:
I don't know much about this file, as it's not really the ospo team's purview. Looking through the blames, it seems like @priscilawebdev has worked on this file a lot, so may be a better person to ask?

Reply:
oh, I'm good. Still working on the tests here and then I add reviewers. Sorry I think your team were just auto-added.